README.md

```diff
@@ -0,0 +1,185 @@
+- **Admin can login, add and delete cars.**
+- **Users can view history of cars**
+- **Users can view characteristics of cars (price, electric, brand and much more).**
+- **Users can view which cars are available for rent and which aren't**
+- **Full stack app**
+- **Styling uses CSS**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+  
+  #### [Pending the live demo](https://pending.onrender.com)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get the content of this project locally you need to run the following commands in your terminal:
+
+and follow these steps.
+
+### Prerequisites
+
+- IDE(code editor) like: **Vscode**, **Sublime**, etc.
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+1. Download the **Zip** file or clone the repo with:
+ ```bash
+git clone git@github.com:gjuliao/car_app_frontend.git
+```
+2. To access cloned directory run:
+```bash
+cd car_app_frontend
+```
+
+### Install
+
+Install dependancies with:
+
+```bash
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```bash
+rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rspec spec
+```
+
+### Deployment
+
+This project would be deployed using:
+
+- [Render](https://render.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Belay Birhanu**
+
+- GitHub: [@belaymit](https://github.com/belaymit)
+
+👤 **Juan Carlos Sanchez**
+
+- GitHub: [@Juank628](https://github.com/Juank628)
+
+👤 **Diego Cuenca**
+
+- GitHub: [@dcuenca](https://github.com/dcuenca)
+
+👤 **Paul Saenz**
+
+- GitHub: [@paulsaenzsucre](https://github.com/paulsaenzsucre)
+
+👤 **Giovanni Juliao**
+
+- GitHub: [@gjuliao](https://github.com/gjuliao)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Add editing features**
+- **Create more users role**
+- **Create admin dashboard**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/gjuliao/car_app_frontend/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank the Microverse team for the creation of this example project.
```

Comment:
We have to give credit to the original designer... "The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project."